ui/widgets/nuxeo-user-suggestion.js

```diff
@@ -232,6 +232,14 @@ import './nuxeo-user-group-formatter.js';
          * Results filtering function (optional).
          */
         queryResultsFilter: Function,
+
+        /**
+         * Set to `true` for invalid.
+         */
+        invalid: {
```

Comment:
is this discussed with PO that we exposing a new property ?

Reply:
removed this changes as its not required